### ISSUE TITLE
Codeigniter sessions with blade

### ISSUE THREAD

**Issue by tapankumar** [coolpraz/php-blade#1]
Can we use Codeigniter sessions with blade with the help of this library?

**Comment by moazamin6** [comment]
@tapankumar yes you can. please visit my library created on `illuminate/view`

You can install the package from composer by running the following command

`composer require moazamin6/creative-blade`

you can also find that package on following link

GitHub: https://github.com/moazamin6/creative-blade
Packagist: https://packagist.org/packages/moazamin6/creative-blade

Thanks
